ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -40,6 +40,34 @@ Verify Ods Users Can Do Http Request That Must Be Redirected to Https
     Should Start With    ${url}    https
     Remove Pipeline Project    project-redirect-http
 
+Verify DSPO Operator Reconciliation Retry
+    [Documentation]    Verify DSPO Operator is able to recover from missing components during the initialization
+    [Tags]      Sanity    Tier1
+    ${local_project_name} =    Set Variable    recon-test
+    ${stopped} =    Set Variable    ${False}
+    # limit is 180 because the reconciliation run every 2 minutes
+    ${timeout} =    Set Variable    180
+    New Project    ${local_project_name}
+    Install DataSciencePipelinesApplication CR    ${local_project_name}    data-science-pipelines-reconciliation.yaml    False
+    # wait the pod be ready
+    Sleep    15s
```

Comment:
can you replace it with `Wait For Pods To Be Ready` keyword from `ods_ci/tests/Resources/OCP.resource?`

Reply:
Fixed. The code is now outdated